### PR TITLE
Update to 0.11.0

### PR DESCRIPTION
**Pull request by kszucs**
Checklist
* [x] Used a fork of the feedstock to propose changes
* [ ] Bumped the build number (if the version is unchanged)
* [x] Reset the build number to `0` (if the version changed)
* [x] [Re-rendered]( https://conda-forge.org/docs/conda_smithy.html#how-to-re-render ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [x] Ensured the license file is being packaged.

### CODE REVIEW

**Comment by kszucs** [comment]
@conda-forge-admin, please rerender

**Comment by kszucs** [comment]
@wesm please review

File: recipe/meta.yaml [hunk 1]
```diff
@@ -26,22 +26,18 @@ requirements:
...
     - setuptools
-    - {{ pin_compatible('numpy', lower_bound='1.10') }}   # [not (win and py>=36)]
-    - {{ pin_compatible('numpy', lower_bound='1.12') }}   # [win and py>=36]
+    - {{ pin_compatible('numpy', lower_bound='1.14') }}
     - pandas
     - six
-    - arrow-cpp 0.10.0
-    - parquet-cpp 1.5.0.pre
     - futures  # [py27]
+    - arrow-cpp {{ version }}
```
Comment (wesm) [inline recipe/meta.yaml]: Do we need to pin the parquet-cpp metapackage? I will merge this as is, but if we need that (to make sure the right binaries get loaded) feel free to open another PR

**Comment by wesm** [comment]
NB, because this had "skip ci" in one of the commits, the squashed commit did not build in Travis/Circle CI. I just pushed a new commit with the skip ci removed

**Comment by kszucs** [comment]
Sorry! I couldn't rerender the feedstock locally, so I wanted to spare the CI resources before requesting the bot.

**Pull request merged**